dll/win32/shell32/wine/shpolicy.c

```diff
@@ -131,10 +131,17 @@ static const char strNoFileMenu[] = {"NoFileMenu"};
 static const char strNoSaveSetting[] = {"NoSaveSettings"};
 static const char strNoClose[] = {"NoClose"};
 static const char strNoRun[] = {"NoRun"};
+static const char strNoSimpleStartMenu[] = {"NoSimpleStartMenu"};
 
 /* policy data array */
 static POLICYDATA sh32_policy_table[] =
 {
+  {
```

Comment:
Keep this sorted by the value of the REST_xxx constant please, i.e. this goes between REST_NONLEGACYSHELLMODE and REST_STARTRUNNOHOMEPATH.